### PR TITLE
Making pyops python3.0 compatible

### PR DESCRIPTION
**Pull request by khabiruddin**
Python has 2.7 outdated, which makes us update all python code compatible to version 3.0.

CCCP-3150

### CODE REVIEW

File: examples/createhostExample.py [hunk 1]
```diff
@@ -26,15 +26,15 @@
 data = opsprod.get_host_by_name("name_of_the_server")
 
 # Print the data for the host
-print opsprod.json_nice(data)
+print (opsprod.json_nice(data))
```
Comment (Aegil) [inline examples/createhostExample.py:29]: Per pep8 standards
you should not have a space before the (
https://www.flake8rules.com/rules/E211.html

Comment (khabiruddin) [reply examples/createhostExample.py]: All the space before print have been removed.

**Comment by junousi** [comment]
When starting with the monitoring task, there was confusion about how to use pyops for cloning hosts.

What do you think, related to this, should we make life easier for the Next Guy? :)

Basically the README.md could be extended to cover both cases 

1) how to set downtime
2) how to use createhostExample.py (or any other alternative solution whose end result is getting a host cloned)

**Comment by khabiruddin** [comment]
> When starting with the monitoring task, there was confusion about how to use pyops for cloning hosts.
> 
> What do you think, related to this, should we make life easier for the Next Guy? :)
> 
> Basically the README.md could be extended to cover both cases
> 
> 1. how to set downtime
> 2. how to use createhostExample.py (or any other alternative solution whose end result is getting a host cloned)

Yes I have made changes in README.md file to show how to add a new node in monitoring.

File: README.md [hunk 2]
```diff
@@ -10,4 +12,19 @@ Usage
...
+
+    $ export prod_ops_user="username"
+    $ read -s prod_ops_pass
+    $ export prod_ops_pass
+    $ export prod_ops_base="https://your-opsview-server/opsview/rest/""    
+    $ vim examples/createhostExample.py
+data = opsprod.get_host_by_name("p5.pouta.csc.fi")
+newserver = { "ip": "p20.pouta.csc.fi", "name": "p20.pouta.csc.fi" }
+
+    $ python pyops/pyops.py examples/createhostExample.py
```
Comment (junousi) [inline README.md:29]: This won't work as python will run only one script from it's input parameters. There are many ways to have python pick up the pyops library, but if you want to do this on a single line you can change this to:

`$ PYTHONPATH=./pyops python examples/createhostExample.py`

File: README.md [hunk 3]
```diff
@@ -10,4 +12,19 @@ Usage
     $ export prod_ops_base="https://your-opsview-server/opsview/rest/""
     $ pyops-downtime -e +1d testserver1 testserver2
 
+Adding node for monitoring:
+
+First we need to edit examples/createhostExample.py file and a add node name from where pyops will fetch template data.
+For example, if you want to add a new nodes in devel environment, then we can take metadata from a devel node p5.pouta.csc.fi.
+If your newserver is p20.pouta.csc.fi, then we can do following.
```
Comment (junousi) [inline README.md:19]: While our hostnames are not really a secret, I think it's good to follow the existing "style" of the repo, which at the moment is quite generic. (Which is good for a public repository).

So if downtime example hosts are `testserver1` and `testserver2`, could you change all these to e.g. `reference-server.domain` & `new-server.domain` or something similarly generic?

File: README.md (hunk 2 above)
Comment (khabiruddin) [reply README.md]: Made changes in README as you have mentioned.

File: README.md (hunk 3 above)
Comment (khabiruddin) [reply README.md]: Made as generic as possible. I have removed all of our secret information from README.

**Pull request merged**